internal/processor/otel/traces.go

```diff
@@ -304,6 +304,8 @@ func TranslateTransaction(
 				netHostPort = int(v.IntVal())
 			case "rpc.grpc.status_code":
 				event.Transaction.Result = codes.Code(v.IntVal()).String()
+			case "grcp.kind":
+				event.Transaction.Type = "request"
```

Comment:
```suggestion
```

I would prefer to minimise non-standard attributes like this, and just stick to those defined by the `rpc` semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#semantic-conventions-for-rpc-spans

Reply:
Again, fair.  This isn't strictly necessary and more a covering our bases sort of thing (and cargo culting other non-standard attributes I saw in this file.  Removing this.